core/frontend/vitest.config.mts

```diff
@@ -0,0 +1,68 @@
+import { coverageConfigDefaults, defineConfig } from 'vitest/config';
+import { viteStaticCopy } from 'vite-plugin-static-copy';
+
+export default defineConfig({
+  test: {
+    dir: "src",
+    setupFiles: "./src/test/setupTests.ts",
+    // include: ["**/<testfileName>.test.ts"], // to adhere to chai's describe.only/it.only
```

Comment:
?

Reply:
people are used to chai's  it.only for filtering tests, when vitest doesn't work that way. This is a placeholder comment for people to use when they want to run just one test file. A long term solution could be exploring using the vitest VS Code extension, in tandem with setting up a vitest workspace for our monorepo